04/app.js

```diff
@@ -2,8 +2,36 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
+        day: '07',
+        month: '06',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+
+// cała ta funkcja działa, wszysto OK,
+// nie mogę tylko skonwertować daty z 0 na początku,
+// kiedy liczba dnia jest jednocyfrowa
```

Comment:
jeśli wartość jest mniejsza niż 10 to trzeba do początku dodać zero np. `(value < 10 ? '0'+value : value)`

Reply:
Możesz wykorzystać tutaj funkcję `startWithZero`

Reply:
poprawiłem.